### ISSUE TITLE
[BUG]when using deepspeed loss learning rate did not down

### ISSUE THREAD

**Issue by AndyLinOuO** [microsoft/DeepSpeed#3958]
**Describe the bug**
i want to train a dolly 2.0 2.8b model with using deepspeed but display on terminal is always same did i miss something?
without using deepspeed it can normal down 

this is using deepspeed when training
![image](https://github.com/microsoft/DeepSpeed/assets/83204959/88c05692-f43a-4e11-9a1b-a72edcd8de90)

this is without using deepspeed
![image](https://github.com/microsoft/DeepSpeed/assets/83204959/626bc724-56a6-40c0-bf6c-2663c0cffc88)

and this is my deepspeed file

{
    "fp16": {
      "enabled": false
    },
    "bf16": {
      "enabled": true
    },
    "optimizer": {
      "type": "AdamW",
      "params": {
        "lr": "auto",
        "betas": "auto",
        "eps": "auto",
        "weight_decay": "auto"
      }
    },
    "scheduler": {
      "type": "WarmupLR",
      "params": {
        "warmup_min_lr": "auto",
        "warmup_max_lr": "auto",
        "warmup_num_steps": "auto"
      }
    },
    "zero_optimization": {
      "stage": 3,
      "overlap_comm": true,
      "contiguous_gradients": true,
      "sub_group_size": 1e9,
      "reduce_bucket_size": "auto",
      "stage3_prefetch_bucket_size": "auto",
      "stage3_param_persistence_threshold": "auto",
      "stage3_max_live_parameters": 1e9,
      "stage3_max_reuse_distance": 1e9,
      "stage3_gather_16bit_weights_on_model_save": true
    },
    "gradient_accumulation_steps": "auto",
    "gradient_clipping": "auto",
    "steps_per_print": 2000,
    "train_batch_size": "auto",
    "train_micro_batch_size_per_gpu": "auto",
    "wall_clock_breakdown": false
  }

its make me so confuse
please help me QQ

**Comment by Ding201901** [comment]
I have the same problem, have you solved it?
